Neos.Neos/Classes/Fusion/Cache/ContentCacheFlusher.php

```diff
@@ -176,6 +191,25 @@ public function shutdownObject()
         }
     }
 
+    /**
+     * @param AssetUsageInNodeProperties $assetUsage
+     * @return ContentContext
+     */
+    protected function getContextForReference(AssetUsageInNodeProperties $assetUsage)
```

Comment:
```suggestion
    protected function getContextForReference(AssetUsageInNodeProperties $assetUsage): ContentContext
```

Reply:
The class has no return type annotation at all, so I was reluctant to add that. But, yeah.